nomic/atlas.py

```diff
@@ -12,10 +12,144 @@
 from loguru import logger
 from tqdm import tqdm
 
-from .project import AtlasProject
+from .dataset import AtlasDataset
 from .settings import *
 from .utils import b64int, get_random_name, arrow_iterator
 
+from .data_inference import NomicProjectHyperparameters, NomicTopicHyperparameters, NomicDuplicatesHyperparameters
+
+
+def map_data(
+    data: List[Dict] = None,
+    embeddings: np.array = None,
+    name: str = None,
+    description: str = "",
+    id_field: str = None,
+    is_public: bool = True,
+    indexed_field: str = None,
+    projection: Optional[Dict] = None,
+    topic_model: Optional[Dict] = None,
+    duplicate_detection: Optional[Dict] = None
```

Comment:
Rather than `None`, what if this was `topic_model: Union[True, False, TopicModelOptions]`. True means build a topic model with default settings: False means don't build a topic model: passing a dict turns it into `**kwargs`.

I never love having `None` mean "do something", and it also keeps the ability to turn off modeling.

Reply:
Done